packages/cypress/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "peerDependencies": {
     "@nrwl/workspace": "*",
-    "cypress": "^3.8.2"
+    "cypress": "^4.1.0"
```

Comment:
We don't think we can have this peer dependency here.  We might want to change the version to * just remove peerDependency altogether. otherwise folks using Cypress 3 will see a warning.

Reply:
I don't think we should remove the peerDependency. I put the dependency as `>= 3 < 5`, instead because it should still say that cypress is a peerDependency.